x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.test.tsx

```diff
@@ -857,37 +870,6 @@ describe('rules_list ', () => {
       expect(screen.queryByTestId('ruleTagFilter')).toBeInTheDocument();
     });
 
-    it('can filter by tags', async () => {
```

Comment:
Deleting this test case because it's timing out, we have a better test in `ftr` for this case anyways.